Big-Spender/readme.md

```diff
@@ -68,39 +68,39 @@ INSERT YOUR QUERY HERE
 **You:** Then here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+INSERT YOUR QUERY HERE: SELECT * FROM spends WHERE description ilike '%fee%';
 ```
 
 **Farnoosh:** Hi, it's me again. It turns out we also need the transactions that have the expense area of 'Better Hospital Food'. Can you help us with that one?
 
 **You:** No worries. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+INSERT YOUR QUERY HERE:  SELECT e.expense_area, s.amount, s.transaction_no FROM spends s JOIN expense_areas e ON e.id = s.expense_area_id WHERE e.expense_area = 'Better Hospital Food';
 ```
 
 **Claire:** Great, that's very helpful. How about the total amount spent for each month?
 
 **You:** You can get that by using the GROUP BY clause. Here's the query:
 
 ```sql
-CREATE YOUR QUERY HERE
+CREATE YOUR QUERY HERE:  SELECT sum(amount) AS total_monthly_amount, date AS month FROM spends GROUP BY month;
```

Comment:
You sure can. I wonder, what other strategy might you use? What about `EXTRACT`?

Reply:
Hi @saqibjvd - I agree with @SallyMcGrath, you should use a date function to get the month from the date but probably not extract.  You've relied on a quirk of the test data that means you only have two dates used (2021-03-01 and 2021-04-01) but you should try to make your code as generic as possible.  If you imagine the data as a much larger realistic set with dates scattered throughout the month and dates covering several years your query won't work correctly.
You should consider using the date_trunc function to get the month value (although this will return a timestamp value for midnight on the first day of the month).  You could also adjust that value to give only the year and month with a bit more work (maybe use extract to get each of those values from the date_trunc result then concatenate them...).

I hope this is helpful.